armi/reactor/tests/test_blocks.py

```diff
@@ -2270,12 +2275,96 @@ def test_gridNotCreatedMultipleMultiplicities(self):
         }
         # add a wire only some places in the block, so grid should not be created.
         wire = components.Helix("wire", "HT9", **wireDims)
-        self.HexBlock.add(wire)
-        self.HexBlock.spatialGrid = None  # clear existing
+        self.hexBlock.add(wire)
+        self.hexBlock.spatialGrid = None  # clear existing
         with self.assertRaises(ValueError):
-            self.HexBlock.autoCreateSpatialGrids()
+            self.hexBlock.autoCreateSpatialGrids()
+
+        self.assertIsNone(self.hexBlock.spatialGrid)
+
+
+class TestHexBlockOrientation(unittest.TestCase):
+    def setUp(self):
+        self.td = TemporaryDirectoryChanger()
+        self.td.__enter__()
+
+    def tearDown(self):
+        self.td.__exit__(None, None, None)
+
+    @staticmethod
+    def getLocalCoordinatesBlockBounds(b: blocks.HexBlock):
+        """Call getLocalCoordinates() for every Component in the Block and find the X/Y bounds."""
+        maxX = -111
+        minX = 999
+        maxY = -111
+        minY = 999
+        for comp in b:
+            locs = comp.spatialLocator
+            if not isinstance(locs, grids.MultiIndexLocation):
+                locs = [locs]
+
+            for loc in locs:
+                x, y, _ = loc.getLocalCoordinates()
+                if x > maxX:
+                    maxX = x
+                elif x < minX:
+                    minX = x
+
+                if y > maxY:
+                    maxY = y
+                elif y < minY:
+                    minY = y
+
+        return minX, maxX, minY, maxY
+
+    def test_validateReactorCornersUp(self):
+        """Validate the spatial grid for a corners up HexBlock and its children."""
```

Comment:
To indicate a point of confusion here: 

1. The reactor is corners up as evidenced by the test name and `self.assertTrue(r.core.spatialGrid.cornersUp)`
2. The comments and docstrings allege a hex block in this reactor will be "corners up"
3. But, we `self.assertFalse(b.spatialGrid.cornersUp)` (which is the correct approach, in my understanding)
4. Later assertions on pin locations reference "flats up block grid"

So there is some ambiguity in what a "flats up" or "corners up" hex block means. Is it the orientation of the bounding hexagonal, derived from the core grid? Or the orientation of the grid attached to the block? I would recommend the former.